starter-project-web-vue2/pages/liben/articles/index.vue

```diff
@@ -0,0 +1,30 @@
+<template>
+  <!-- <div> -->
+  <v-container>
+    <v-row no-gutters>
+      <v-col v-for="blog in blogs" :key="blog.id" cols="4">
+        <LibenArticle :blog="blog" />
+      </v-col>
+    </v-row>
+  </v-container>
+  <!-- </div> -->
+</template> 
+
+<script>
+import { mapState } from 'vuex'
```

Comment:
Its better to use mapGetters instead. since the getters are already implemented.